python/lsst/ap/association/packageAlerts.py

```diff
@@ -82,6 +82,12 @@ class PackageAlertsConfig(pexConfig.Config):
         default=False,
     )
 
+    doWriteFailedAlerts = pexConfig.Field(
+        dtype=bool,
+        doc="Write alerts which fail to send to disk for debugging purposes.",
```

Comment:
It's probably worth mentioning that this option is only used if `doProduceAlerts`, and is independent of `doWriteAlerts`.

Reply:
I find the wording a bit hard to parse; maybe:
```suggestion
        doc="If an alert cannot be sent, write it to disk for debugging purposes.",
```